pkg/asset/installconfig/basedomain.go

```diff
@@ -57,7 +57,12 @@ func (a *baseDomain) Generate(parents asset.Parents) error {
 		return platform.Azure.SetBaseDomain(zone.ID)
 	case gcp.Name:
 		var err error
-		a.BaseDomain, err = gcpconfig.GetBaseDomain(platform.GCP.ProjectID)
+		zone, err := gcpconfig.GetBaseDomain(platform.GCP.ProjectID)
+		if err != nil {
+			return err
+		}
+		a.BaseDomain = zone.DNSName
+		platform.GCP.PublicZoneID = zone.ID
```

Comment:
Similar comment to https://github.com/openshift/installer/pull/2764/commits/0867f8d42a8506069762981bc60abec48202f38d#r386540128